genetics_ark/templates/500.html

```diff
@@ -0,0 +1,173 @@
+{% load static %}
```

Comment:
what are these for?

Reply:
to load the files in staticfiles, 500.html need it for the font (FiraMono-Medium.ttf)

Reply:
sorry I meant more generally the new 404 and 500 templates, what do they look like?

Reply:
![Screenshot from 2022-07-27 09-02-23](https://user-images.githubusercontent.com/60819172/181194638-c55a74da-c9cf-4410-abc3-c6fb299012bc.png)

Reply:
Same image, but different errors: 404 Error, 500 Error & 502 Error. The "HOME" points to `/`